### PR TITLE
Rentify 14 create profile page

### PR DESCRIPTION
**Pull request by Slavchev03**

### CODE REVIEW

File: back-end/src/main/java/com/devminds/rentify/config/AuthConfig.java [hunk 1]
```diff
@@ -50,6 +50,10 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                                 .requestMatchers("/rentify/likes/**").permitAll()
                                 .requestMatchers("/rentify/pictures").permitAll()
                                 .requestMatchers("/rentify/pictures/**").permitAll()
+
+                                .requestMatchers("/rentify/**").permitAll()
+                                .requestMatchers("/**").permitAll()
```
Comment (ValentinTsonevskii) [inline back-end/src/main/java/com/devminds/rentify/config/AuthConfig.java:55]: use specific path

File: back-end/src/main/java/com/devminds/rentify/entity/Address.java [hunk 2]
```diff
@@ -1,11 +1,6 @@
 package com.devminds.rentify.entity;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```
Comment (maria-markova) [inline back-end/src/main/java/com/devminds/rentify/entity/Address.java:3]: remove wildcard import

File: front-end/src/components/items/ItemsList.jsx [hunk 3]
```diff
@@ -44,6 +41,7 @@ const ItemsList = (notShowDropdown ,categoryId) => {
               : "/category/" + endpointSuffix.id)
         );
         setItems(result);
+        console.log(result);
```
Comment (maria-markova) [inline front-end/src/components/items/ItemsList.jsx:44]: remove console.logs

**Review by aleksandramihaylova — APPROVED** [review]

File: back-end/src/main/java/com/devminds/rentify/controller/UserController.java [hunk 4]
```diff
@@ -1,15 +1,12 @@
...
 import com.devminds.rentify.service.UserServiceImpl;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (maria-markova) [inline back-end/src/main/java/com/devminds/rentify/controller/UserController.java:8]: remove wildcard import pls

**Review by KalinVasilev-at-blankfactor — APPROVED** [review]

File: back-end/src/main/java/com/devminds/rentify/service/UserServiceImpl.java [hunk 5]
```diff
@@ -12,17 +18,18 @@
...
 import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.io.IOException;
+import java.net.URL;
 import java.security.GeneralSecurityException;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```
Comment (maria-markova) [inline back-end/src/main/java/com/devminds/rentify/service/UserServiceImpl.java:32]: remove wildcard import

**Pull request merged**